`AppAccessSection › editing` flakiness

## Before submitting a new issue

## Failure

#### Link(s) to logs

- https://github.com/gravitational/teleport/actions/runs/13574510727/job/37947342763

#### Relevant snippet

```
Summary of all failing tests
FAIL web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.test.tsx (16.119 s)
  ● AppAccessSection › editing

    thrown: "Exceeded timeout of 5000 ms for a test.
    Add a timeout value to this test to increase the timeout, if this is a long-running test. See https://jestjs.io/docs/api#testname-fn-timeout."

      311 |     within(gcpServiceAccounts()).getAllByRole('textbox');
      312 |
    > 313 |   test('editing', async () => {
          |   ^
      314 |     const { user, onChange } = setup();
      315 |     await user.click(screen.getByRole('button', { name: 'Add a Label' }));
      316 |     await user.type(screen.getByPlaceholderText('label key'), 'env');

      at test (web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.test.tsx:313:3)
      at Object.describe (web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.test.tsx:282:1)


Seed:        -1503809771
Test Suites: 1 failed, 288 passed, 289 total
Tests:       1 failed, 1667 passed, 1668 total
Snapshots:   1 passed, 1 total
Time:        103.159 s
Ran all test suites.
 ELIFECYCLE  Test failed. See above for more details.
Error: Process completed with exit code 1.
```

--- a/web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.test.tsx
+++ b/web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.test.tsx
@@ -315,27 +315,28 @@ describe('AppAccessSection', () => {
     await user.click(screen.getByRole('button', { name: 'Add a Label' }));
     await user.type(screen.getByPlaceholderText('label key'), 'env');
     await user.type(screen.getByPlaceholderText('label value'), 'prod');
+
+    // Instead of typing these ungodly long values, we paste them — otherwise,
+    // this test may time out. And that's what our users would typically do,
+    // anyway.
     await user.click(
       within(awsRoleArns()).getByRole('button', { name: 'Add More' })
     );
-    await user.type(
-      awsRoleArnTextBoxes()[1],
-      'arn:aws:iam::123456789012:role/admin'
-    );
+    await user.click(awsRoleArnTextBoxes()[1]);
+    await user.paste('arn:aws:iam::123456789012:role/admin');
     await user.click(
       within(azureIdentities()).getByRole('button', { name: 'Add More' })
     );
-    await user.type(
-      azureIdentityTextBoxes()[1],
+    await user.click(azureIdentityTextBoxes()[1]);
+    await user.paste(
       '/subscriptions/1020304050607-cafe-8090-a0b0c0d0e0f0/resourceGroups/example-resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/admin'
     );
     await user.click(
       within(gcpServiceAccounts()).getByRole('button', { name: 'Add More' })
     );
-    await user.type(
-      gcpServiceAccountTextBoxes()[1],
-      'admin@some-project.iam.gserviceaccount.com'
-    );
+    await user.click(gcpServiceAccountTextBoxes()[1]);
+    await user.paste('admin@some-project.iam.gserviceaccount.com');
+
     expect(onChange).toHaveBeenLastCalledWith({
       kind: 'app',
       labels: [{ name: 'env', value: 'prod' }],

